UnityProject/Assets/Scripts/Camera/CameraEffectControlScript.cs

```diff
@@ -27,5 +32,24 @@ public void ToggleNightVisionEffectState()
 		{
 			nightVisionCamera.enabled = !nightVisionCamera.enabled;
 		}
+
+		private void Update()
```

Comment:
maybe better served to use the update manager?

Reply:
changed.